WalletWasabi/Wallets/Exchange/ExchangeRateProvider.cs

```diff
@@ -78,7 +78,11 @@ private static async Task<ExchangeRate> GetExchangeRateAsync(string providerName
 		return Result<decimal, Exception>
 			.Catch(() => extractor(json))
 			.Match(
-				rate => new ExchangeRate("USD", rate),
+				rate =>
+				{
+					Logger.LogInfo($"Fetched exchange rate from {providerName}: {rate}.");
```

Comment:
```suggestion
					Logger.LogInfo($"Fetched exchange rate from {providerName}: {rate} USD.");
```
nit, I think it's better to mention